R/FilteredDatasetDataframe.R

```diff
@@ -115,19 +118,19 @@ DataframeFilteredDataset <- R6::R6Class( # nolint
     },
 
     #' @description
-    #' Gets the filter expression.
+    #' Gets the subset expression.
     #'
-    #' This functions returns filter calls equivalent to selected items
-    #' within each of `filter_states`. Configuration of the calls is constant and
+    #' This function returns subset expressions equivalent to selected items
+    #' within each of `filter_states`. Configuration of the expressions is constant and
     #' depends on `filter_states` type and order which are set during initialization.
-    #' This class contains single `FilterStates`
-    #' which contains single `state_list` and all `FilterState` objects
-    #' applies to one argument (`...`) in `dplyr::filter` call.
+    #' This class contains single `FilterStates` which contains single `state_list`
+    #' and all `FilterState` objects apply to one argument (`...`) in a `dplyr::filter` call.
     #'
     #' @param sid (`character`)
-    #'  when specified then method returns code containing filter conditions of
-    #'  `FilterState` objects which `"sid"` attribute is different than this `sid` argument.
+    #'  when specified, the method returns code containing conditions calls of
+    #'  `FilterState` objects with `sid` different to that of this `sid` argument.
     #'
+    # TODO verify
```

Comment:
!

Reply:
I added three, all in cases where I didn't feel comfortable making the necessary change myself.

Reply:
One more thing, some TODOs predate this PR and are will not be tackled here.

Reply:
https://github.com/insightsengineering/teal.slice/issues/542
https://github.com/insightsengineering/teal.slice/issues/543